build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
```

Comment:
I like the idea of providing a pandoc command that generates docx if we can get a reasonable docx version.  I don't have a strong opinion about where that command should go.

If we don't track the docx files in the repo, a user will have to be more computationally savvy to use them.  It's easy for us to run the build script from a particular point in the history, but less-computational users may not be able to.  I'm not sure whether to tradeoff storage for ease of export until we know the target audience better.